Releasenotes.md

```diff
@@ -1,5 +1,80 @@
 # Release notes BRK-Bevragen
 
+
+## **Versie 1.3.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Nieuwe endpoints toegevoegd voor stukken en stukdelen.
+  - Bestaande resources zijn (voorzover van toepassing) uitgebreid met relaties naar de betrokken stukken en stukdelen.
+
+- stukken
+  2 nieuwe endpoints:
+  - /stukken/{stukidentificatie}
+  - /stukdelen/{stukdeelidentificatie}
+
+- kadastraalOnroerendeZaken
+  - nummeraanduidingIdentificatie is toegevoegd als queryparameter aan /kadastraalonroerendezaken
+  - Properties toegevoegd:
+    - indicatieVervallenKadastraalOnroerendeZaak
+    - einddatumRecht
+    - isOvergegaanIn
+    - isOntstaanUit
+    - bijbeherendeGrondperceelIdentificatie
+    - bijbehorendeAppartementsrechtIdentificaties
+    - stukIdentificaties
+    - adresseerbaarObjectIdentificatie (bij adres)
+  - Links toegevoegd:
+    - isOntstaanUit (achterwaartse filiatie naar gerelateerde kadastraalOnroerendeZaak)
+    - isOvergegaanIn (voorwaartse filiatie naar gerelateerde kadastraalOnroerendeZaak)
+    - stukken
+    - adressen ([Haal-Centraal BAG-endpoint](https://vng-realisatie.github.io/Haal-Centraal-BAG-bevragen/) is nu in productie beschikbaar )
+    - adresseerbareObjecten, ook beschikbaar via [Haal-Centraal BAG-endpoint](https://vng-realisatie.github.io/Haal-Centraal-BAG-bevragen/)
```

Comment:
hoe bedoel je "ook beschikbaar via"?

Reply:
Net zoals de adressen die ik in de regel daarboven genoemd heb. ALs je het onduidelijk vind wil ik het ook anders formuleren. Geef maar aan.

Reply:
iets als:
>     - adressen (link naar [Haal-Centraal BAG adressen endpoint](https://vng-realisatie.github.io/Haal-Centraal-BAG-bevragen/) is nu in productie beschikbaar )
>     - adresseerbareObjecten (link naar [Haal-Centraal BAG adresseerbareobjecten endpoint](https://vng-realisatie.github.io/Haal-Centraal-BAG-bevragen/))